components/trace/appmap/index.test.mjs

```diff
@@ -213,7 +213,7 @@ assertDeepEqual(
           thread_id: 0,
           defined_class: "filename",
           method_id: "f",
-          path: "dirname/filename.js",
+          path: "protocol://host/home/dirname/filename.js",
```

Comment:
The commit message mentions

> The relative representation if it exists is still preferred

and also

> paths in the appmaps are changed from relative to absolute

How does this change affect end-users?